testr/testr.go

```diff
@@ -67,6 +87,44 @@ type Underlier interface {
 	GetUnderlying() *testing.T
 }
 
+// UnderlierInterface exposes access to the underlying TestingT instance. Since
+// callers only have a logr.Logger, they have to know which
+// implementation is in use, so this interface is less of an
+// abstraction and more of a way to test type conversion.
+type UnderlierInterface interface {
+	GetUnderlying() TestingT
+}
+
+func logInfo(
```

Comment:
Can you please un-linewrap the args?  This is inconsistent with anything else in the codebase

Reply:
Sorry for that.